features/zoeken/met-geslachtsnaam-voornamen-en-gemeente-van-inschrijving/overzicht.feature

```diff
@@ -0,0 +1,73 @@
+# language: nl
+Functionaliteit: zoek personen met geslachtsnaam, voornamen en gemeente van inschrijving
+
+  Als consumer
+  wil ik personen kunnen vinden door het opgeven van de gemeente waar ze zijn ingeschreven én hun geslachtsnaam en voornamen
+
+  Achtergrond:
+    Gegeven adres 'A1'
+      | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) |
```

Comment:
waarom wordt hier identificatiecode verblijfplaats (11.80) opgegeven? Doen we niks mee in deze feature

Reply:
Melvin verwijdert kolom identificatiecode verblijfplaats (11.80)

Reply:
done